src/vtp/create_blank_ballot.py

```diff
@@ -27,16 +27,16 @@
 
 # Standard imports
 # pylint: disable=wrong-import-position
-import sys
 import argparse
 import logging
-from logging import info, debug
 import pprint
+import sys
+from logging import debug, info
 
 # Local imports
-from address import Address
-from ballot import BlankBallot
-from election_config import ElectionConfig
+from utils.address import Address
+from utils.ballot import BlankBallot
+from utils.election_config import ElectionConfig
```

Comment:
Ditto: use relative imports:

```suggestion
from .utils.address import Address
from .utils.ballot import BlankBallot
from .utils.election_config import ElectionConfig
```